client/src/app/core/core-services/notify.service.ts

```diff
@@ -75,12 +75,13 @@ interface ChannelIdResponse {
     channel_id: string;
 }
 
-const IccPath = '/system/icc';
+const notifyPath = '/system/icc/notify';
+const publishPath = `${notifyPath}/publish`;
 /**
  * Does currently not exist
  */
 // const iccHealthPath = `${IccPath}/health`
```

Comment:
Should this line not be removed?

Reply:
no, just have to be adjusted when the health path exist